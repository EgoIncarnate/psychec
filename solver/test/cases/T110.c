void
usage ()
{
    printf (_("\
Usage: %s [OPTION]... [FILE]\n\
Base%d encode or decode FILE, or standard input, to standard output.\n\
"), program_name, BASE_TYPE);

}
