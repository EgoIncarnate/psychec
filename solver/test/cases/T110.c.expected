int BASE_TYPE ;
char* _ (char const*) ;
char* program_name ;
